### ISSUE TITLE
Alt text on images

### ISSUE THREAD

**Issue by alexy4744** [QuestPDF/QuestPDF#1102]
**Is your feature request related to a problem? Please describe.**
We should be able to specify the alt text on images, so that the PDFs can be accessible to screen readers.

**Describe the solution you'd like**
A method like `.Alt()` that chains with `.Image()` to allow emitting alt text associated to that image in the output pdf.

```csharp
  column
        .Item()
        .Image("sunset.jpg")
        .Alt("Sunset over a lake with a mountain in the distance with vibrant orange and purple hues.")
```

**Comment by MarcinZiabek** [comment]
Thank you for this suggestion. I totally agree! ❤  

Accessibility is a crucial aspect of document generation. I’ve added your suggestion to the roadmap and plan to begin researching and implementing Tagged PDF support in the coming months.
